examples/aggregator/example.toml

```diff
@@ -0,0 +1,9 @@
+name = "aggregator"
+
+[modules.module]
+lang = "Rust"
```

Comment:
You probably don't need both the lang and cargo_manifest? If it has a cargo_manifest it should be clear it's cargo :)

Reply:
In pure Rust this would be an enum with different fields, and everything would be type safe. In principle, that could be done for TOML too, but the parser does not seem to like those kind of enums :/ https://github.com/alexcrichton/toml-rs/issues/390